### DISCUSSION TITLE
Ability to create parkour kit

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#353] [To-do List]
Status: Not started (6/6/2023)

**Comment by liexes** [comment]
I did this mode, it works with a single plate or well, with a custom item that the user prefers, when reaching it, the one that the floor wins, therefore it would only serve in ffa, it would also serve in party split but kill the team too (I have no placeholders to manage this)

**Comment by liexes** [comment]
I am willing to pass the code, is 100% customizable ( I can pass the instructuons for install only is 3)

**Comment by minehostil** [comment]
> I am willing to pass the code, is 100% customizable ( I can pass the instructuons for install only is 3)

you can send it ?  it looks quite interesting

**Comment by HierarchyMan** [comment]
> > I am willing to pass the code, is 100% customizable ( I can pass the instructuons for install only is 3)
> 
> you can send it ?  it looks quite interesting

I made a work on progress addon with as many checkpoints u want and victory
No setup required, test video on unofficial official eden discord #sneak-peaks

**Reply by HierarchyMan** [reply]
Mine works in duels too btw
And soon I'm gonna be utilising eden events to make it work with match start and match end events

**Reply by HierarchyMan** [reply]
@Antoniova1 
https://dsc.gg/edensupport

**Reply by minehostil** [reply]
Ok thx
